### PR TITLE
Add isUserAdminOfCurrentProject field to application context

### PR DESCRIPTION
**Pull request by obulaworld**
#### Summary

This PR adds isUserAdminOfCurrentProject to application-context.

### CODE REVIEW

File: packages/application-shell/src/test-utils/test-utils.tsx [hunk 1]
```diff
@@ -96,7 +96,6 @@ const defaultUser = {
   launchdarklyTrackingSubgroup: 'dev',
   launchdarklyTrackingId: '111',
   launchdarklyTrackingTeam: undefined,
-  launchdarklyTrackingTenant: 'gcp-eu',
```
Comment (tdeekens) [inline packages/application-shell/src/test-utils/test-utils.tsx:99]: Why did you remove this?

Comment (obulaworld) [reply packages/application-shell/src/test-utils/test-utils.tsx:99]: It is deprecated and was causing the test to fail. I guess it has been replaced by launchdarklyTrackingCloudEnvironment.

**Comment by CarlosCortizasCT** [comment]
Hello,

I pushed [this commit](https://github.com/commercetools/merchant-center-application-kit/pull/3523/commits/7a9cf21c9a188b2ef744a58c3549d67de091b1d5) to include refreshed types and schemas from the backend services as I found out this helps with the error we were having (although I'm not sure why).

**Review by CarlosCortizasCT — APPROVED** [review]
Thanks! 👍

**Review by tdeekens — APPROVED** [review]

**Review by emmenko — APPROVED** [review]
Thanks!
